opty/direct_collocation.py

```diff
@@ -573,6 +573,17 @@ def plot_constraint_violations(self, vector, axes=None):
             The initial guess, solution, or any other vector that is in the
             canonical form.
 
+        subplots : boolean, optional.
+            If True, the equations of motion will be plotted in a separate
+            plot for each equation of motion.
+            Default is False.
+            If a user wants to provide the axes, it is recommended to run once
+            without providing axes, to see how many are needed.
+
+        axes : ndarray of AxesSubplot, optional.
+            If given, it is the user's responsibility to provide the correct
+            number of axes.
```

Comment:
These are not in the same order as the function signature and there should not be an empty line between kwargs in the docstring.